### ISSUE TITLE
Abbreviations are stripped from the left pane table of contents

### ISSUE THREAD

**Issue by paralexm** [retypeapp/retype#617]
Consider two articles with abbreviations in their titles:

- CSV File
- TSV File

In the left pane ToC, they correctly appear with their full titles:
![image](https://github.com/retypeapp/retype/assets/87371546/ff18cc25-b899-4fe6-8f12-4dcc2e4c7c1a)

Now if we add the acronyms in the `bottom.md` template like this:
```md
*[CSV]: Comma Separated Values
*[TSV]: Tab Separated Values
```

Now the left pane ToC will show titles: `File` and `File`:
![image](https://github.com/retypeapp/retype/assets/87371546/df43759e-34d6-43e1-99bf-6529f10a9be7)

**Comment by geoffreymcgill** [comment]
Thanks for the report. This is a defect and we will try to fix right away. 

For anyone interested in this topic, the following demonstrates adding project-wide abbreviations:

https://github.com/retypeapp/retype/blob/main/_includes/bottom.md?plain=1

**Comment by geoffreymcgill** [comment]
This defect has been fixed and will be included in the upcoming Retype v3.6 release. 

<img width="311" alt="Screen Shot 2023-10-31 at 11 54 50 AM" src="https://github.com/retypeapp/retype/assets/62210/0959862d-5399-4217-bf1d-2fc0696bac9d">

@paralexm Thanks again for reporting!
